src/app.py

```diff
@@ -338,13 +343,15 @@ def upload_csv():
         )
 
     # make sure these columns aren't empty
-    if data[["uuid", "name", "sealed_state"]].isna().sum().sum() > 0:
-        logger.error("uuid or name or sealed_state column values are empty!")
+    if data[CONSTANTS.CATALOGUE_POST_MANDATORY_FIELDS].isna().sum().sum() > 0:
+        logger.error(
+            CONSTANTS.ERROR_MSG_ANY_OF_THE_CATALOGUE_POST_MANDATORY_FIELDS_EMPTY
```

Comment:
It's not in the scope of constants now, right?

Reply:
yes Iksha, fixed and tested it.